sycl/test/lit.cfg.py

```diff
@@ -41,6 +41,8 @@
 # Configure LD_LIBRARY_PATH or corresponding os-specific alternatives
 if platform.system() == "Linux":
     config.available_features.add('linux')
+    if config.sycl_use_libcxx == "ON":
+        config.available_features.add('libcxx')
```

Comment:
It looks like libcxx has an experimental support on Windows as well. I think LIT scripts shouldn't limit libcxx configuration to Linux platform only.

Reply:
Hi, @bader 
I haven't tried to build sycl runtime using libcxx on Windows, @alexbatashev , Is the experimental feature(building libcxx with llvm libcxx) supported on Windows?
Thanks very much.

Reply:
I have never tried it on Windows. libcxx.llvm.org does not mention Windows as one of the supported platforms, though.

Reply:
I was using this document: https://libcxx.llvm.org/docs/BuildingLibcxx.html#experimental-support-for-windows

Reply:
Hi, @bader and @alexbatashev 
If we want to enable building SYCL runtime with llvm libcxx on Windows, we need to 'use' libc++ with MSVC CL compiler on Windows which means we need to build all the SYCL runtime using MSVC CL compiler and use llvm libc++ instead of MSVC's C++ std library meanwhile.
According to the document: https://libcxx.llvm.org/docs/UsingLibcxx.html
We can find how to 'use' libc++ together with g++ or clang++ on Linux but can't find any information about using MSVC CL compiler with libc++.
Thanks very much.

Reply:
The point of my question is: should we block testing with libc++ on Windows in lit.cfg.py?
If so, could add a comment to the script explaining why this feature is not available on Windows platform.

Reply:
Hi, @bader 
I added comments to explain whey we block testing with libc++ on Windows as MSVC CL compiler doesn't support to use llvm libc++ instead of MSVC C++ std library.
Thanks very much.